ops/terraform/services/base/values/prod.yaml

```diff
@@ -44,32 +44,21 @@
-/bfd/${env}/server/sensitive/data_server_new_relic_metric_key: <<CIPHER>>AgV4QvJg75UuTMy2JdRXmJJLlmb-jbJbgTBCHN9-BPsmrUEAdAACAAtBcHBsaWNhdGlvbgAGY2lwaGVyABVhd3MtY3J5cHRvLXB1YmxpYy1rZXkAREFtWWREYURlOFdnUzI3VVhUNStjL1BKZW12UEU2dDQ5M0RCcERRN0FDemJCRG9mbDc2OVVlaVlxUlRsRGE3RnhFZz09AAEAB2F3cy1rbXMAS2Fybjphd3M6a21zOnVzLWVhc3QtMTo1NzczNzM4MzE3MTE6a2V5L21yay1iMjNlYjZmZWVlYmM0MzQ2YTk5YTZhNTliYzU2MWRmMwC4AQIBAHipsbBUtIKaXI95_AIHiGnjDkxfPDRzgqb6WVwMZI7-kwF-Gt43yFteRGG25ucAzuUKAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMcV_tgDQ1RV7VTFRmAgEQgDthJuRSK8Mkr_aIZo75NlUAKatU5zmCV9Eqpes1D0O58yORs4fpg9o-B4e5JjL2e5sY7l9_nVbFgTc2LAIAABAAax2fe1rmitKNgEzdUyCW50SIFdj67yZFtxGUiTBwxYO7QhVLoK0I4nHIbq93hQXM_____wAAAAEAAAAAAAAAAAAAAAEAAAAl4LGfY6Cr0-wVN8T65FQHXdq41IJfBc3f5mjTs14s7RJ11eGcHaN3WmyC1UztU334eJZV5LoAZzBlAjEA13NtGRM_kUu56Qli5G0qe7frmxOOqv4tj40h4BGzMQn4a0VLXiAnMsQFF2IIE8ITAjAhB8qgsnjsVArkob4PRX124hR4bEIRq3c5H906wwASgnO1BYMAEWKAJAtdV8ZI85M=<</CIPHER>>
 /bfd/${env}/server/sensitive/db/password: <<CIPHER>>AgV4fSmne_KaSnRT5K5zO2HY187JIUuY74JYKvITXGXqXREAdAACAAtBcHBsaWNhdGlvbgAGY2lwaGVyABVhd3MtY3J5cHRvLXB1YmxpYy1rZXkAREEwWVRDMXpkZ0NCWE9JdU9TK3NyRXZYVC80ZXlJdVV0Uk5RSFBXajZmaUVVYzY1NmhyYldVZ1JmUGFsUXd0Y2NSQT09AAEAB2F3cy1rbXMAS2Fybjphd3M6a21zOnVzLWVhc3QtMTo1NzczNzM4MzE3MTE6a2V5L21yay1iMjNlYjZmZWVlYmM0MzQ2YTk5YTZhNTliYzU2MWRmMwC4AQIBAHipsbBUtIKaXI95_AIHiGnjDkxfPDRzgqb6WVwMZI7-kwGISr7_lBP4aZagQfivb1ErAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMEBT09Kitnb6rMMHlAgEQgDuEXO2OB07twL8xKcQDthnadgOQvAzqwrr0wpzalksaiMj1cVw8K96SoAxzQXh85dGBtChII_CWU1TshwIAABAAj9VOqBuG8zSDw-WnjBpLAwuJuOasd4dENXuDneAqfqcHYxNJcu9-lCkLHlFJ6TN8_____wAAAAEAAAAAAAAAAAAAAAEAAAAYhCv5nXMENi1tqgz66iE6KFgXbZU8YQMlDFxNStYAZu8Me_vRJDL-hgBnMGUCMHhUCDJzArlowudSW1ui8PpF6WzJDAHq7zw0MwBam2m13c7-NwNimTvZJ7qmeoPcIQIxAJyYNHwW4ieimjq4IUhXHgx0eRxvhPcdX-m0D7qBhRq-8UlJgShDHt8i1ZX3Tdwl9Q==<</CIPHER>>
-/bfd/${env}/server/sensitive/db/url: <<CIPHER>>AgV4pjmCSi3RUOFjG5TJIe9F9AdFOQZLVry6Y1qHXD83CTAAdAACAAtBcHBsaWNhdGlvbgAGY2lwaGVyABVhd3MtY3J5cHRvLXB1YmxpYy1rZXkAREFyQUNHTURHendTcVVjQXA4UVRubFdYVVB5WERkeHpnUytWbjU3cDVoSExWUnhMeWViYmJvMkxSakM0eGlFN3dCQT09AAEAB2F3cy1rbXMAS2Fybjphd3M6a21zOnVzLWVhc3QtMTo1NzczNzM4MzE3MTE6a2V5L21yay1iMjNlYjZmZWVlYmM0MzQ2YTk5YTZhNTliYzU2MWRmMwC4AQIBAHipsbBUtIKaXI95_AIHiGnjDkxfPDRzgqb6WVwMZI7-kwFRinwW-Gcgjvl36g1oIaoMAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMi7vNiWfFx8B-JvcDAgEQgDtPfB1G_yN1GyBFredKuEClnnnp2F8QNWY6o66A5YwzaDMGJ6CFBoyWYVKC3aoVn7YwJWdAlakPE5THvwIAABAAu3JZa7_3rbFv6xaSqmefIms2JFONJI2ic3AnHOlRHrjLIBeFgLGCLE3iHaWhFZ6u_____wAAAAEAAAAAAAAAAAAAAAEAAACEdaarQVDrJgQmnwo168Y1xNna-UXLNzYUiiy3ZVm48vjrHhK0cGe4B8wIcgG0_ubLWiaDm-zlE9fEV--lqREDQe8ps2ZNyQP4fTZkz21tYlwY5MD-yXDBPhV2Q3sZQ_sBerNy6HuBgh9EskFneW5ekZIflZ7k1t-6R8QBgjBrpCop7PyjcTEV7n2d7SXjHUbYwo3dlgBnMGUCMHVDCBnHym4coBoAj0ANp8UmFVQCP0cVzQT-4Fka3sG_caHo-vlUHzLl02Km2exbgQIxAO7Htm-uJX0M88H5IUjFqFSvL6DIkAdlgKdNNS1eMrmruyupqQHVoajARZUIv5ZWjA==<</CIPHER>>
```

Comment:
Seems like there's still code that references `db/url`. Is that getting set elsewhere now?